Formula/molecule.rb

```diff
@@ -264,23 +274,43 @@ class Molecule < Formula
   end
 
   resource "websocket-client" do
-    url "https://files.pythonhosted.org/packages/bc/a6/5ef697df40e56459de75b57f302f2da92a51269925282a08e4eb76b16fb6/websocket-client-1.1.1.tar.gz"
-    sha256 "4cf754af7e3b3ba76589d49f9e09fd9a6c0aae9b799a89124d656009c01a261d"
+    url "https://files.pythonhosted.org/packages/4e/8f/b5c45af5a1def38b07c09a616be932ad49c35ebdc5e3cbf93966d7ed9750/websocket-client-1.2.1.tar.gz"
+    sha256 "8dfb715d8a992f5712fff8c843adae94e22b22a99b2c5e6b0ec4a1a981cc4e0d"
   end
 
   def install
     virtualenv_install_with_resources
   end
 
+  def caveats
+    <<~EOS
+
+      Depending on the Molecule driver chosen, you may need to install additional packages.
+
+      Molecule-Docker Driver
+      ----------------------
+        - Docker
+        - community.docker Ansible collection
+
+      Molecule-Vagrant Driver
+      -----------------------
+        - VirtualBox
+        - Vagrant
+
+    EOS
+  end
```

Comment:
Is this not in upstream documentation?

Reply:
The general comment about "you may need additional packages depending on the driver" is mentioned in the [Molecule Docs](https://molecule.readthedocs.io/en/latest/installation.html#requirements) but specific requirements for the Vagrant & Docker drivers are scattered throughout those specific project pages outside the main docs.  

I figured a "Caveats" section was appropriate since Docker, VirtualBox, & Vagrant weren't hard dependencies in the formula but are required to use the included drivers.  Additionally, we couldn't include the community.docker Ansible collection as a formula dependency but it is also a manually required setup step by the user to use the Docker driver.

Do you have a suggestion on how this should be best addressed?  I feel something needs to be mentioned in Caveats to alert the user that full use of the formula functionality requires additional steps.  I felt putting the list of dependencies in the Caveats makes it easiest for the user to identify requirements and quickly install them if needed.

Reply:
This is best addressed in upstream documentation. If it's not easy to find there, let's help them clean it up.

Each formula needs to be maintainable by contributors who don't use the formula (and know only how to use Homebrew), and adding `caveats` like this gets in the way of that, as one needs to actually know how to use Molecule to understand what this says and whether it still applies.

Reply:
As a rule, caveats are only to notify users of Homebrew inconsistencies compared to default upstream documentation.